src/os/nusys/nuContInit.c

```diff
@@ -0,0 +1,12 @@
+#include "common.h"
+#include "nu/nusys.h"
+
+u8 nuContInit(void) {
+    u8 contBits;
```

Comment:
small thing but I prefer the initialization to happen on the same line it's declared if possible (as is in this case)